package.json

```diff
@@ -27,6 +27,7 @@
     "docs": "npm run docs:lint && npm run docs:api",
     "jsdoc": "jsdoc",
     "docs:api": "jsdoc -c .jsdoc.json",
+    "postdocs:api": "node ./build/fix-api-docs.js",
```

Comment:
Should run directly after `docs:api` and before anything else I would think. Would we rather use `npm-run-all` here so its more clear?